From c6ae12fd7c560c2cfbc9cb695c29ac1a36e072cd Mon Sep 17 00:00:00 2001
From: Jan Chaloupka <jchaloup@redhat.com>
Date: Fri, 20 Nov 2015 14:36:15 +0100
Subject: [PATCH] add missing short option for --server

---
 pkg/cmd/util/clientcmd/clientconfig.go | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/pkg/cmd/util/clientcmd/clientconfig.go b/pkg/cmd/util/clientcmd/clientconfig.go
index c152100..64ceedf 100644
--- a/pkg/cmd/util/clientcmd/clientconfig.go
+++ b/pkg/cmd/util/clientcmd/clientconfig.go
@@ -18,6 +18,10 @@ func DefaultClientConfig(flags *pflag.FlagSet) clientcmd.ClientConfig {
 	overrideFlags.ContextOverrideFlags.Namespace.ShortName = "n"
 	overrideFlags.AuthOverrideFlags.Username.LongName = ""
 	overrideFlags.AuthOverrideFlags.Password.LongName = ""
+
+	// for compatibility with kubernetes, it is still provided
+	overrideFlags.ClusterOverrideFlags.APIServer.ShortName = "s"
+
 	clientcmd.BindOverrideFlags(overrides, flags, overrideFlags)
 	cobra.MarkFlagFilename(flags, overrideFlags.AuthOverrideFlags.ClientCertificate.LongName)
 	cobra.MarkFlagFilename(flags, overrideFlags.AuthOverrideFlags.ClientKey.LongName)
-- 
1.9.3

